smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -202,6 +222,38 @@ async Task<int> OnExecuteAsync()
                 string connectionString = this.IotHubConnectionString ??
                                           await SecretsHelper.GetSecretFromConfigKey("iotHubConnStrKey");
 
+                Option<DPSAttestation> dpsAttestation = Option.None<DPSAttestation>();
+                if (!string.IsNullOrEmpty(this.DPSScopeId))
+                {
+                    if (string.IsNullOrEmpty(this.DPSEndpoint))
+                    {
+                        throw new ArgumentException("DPS Endpoint cannot be null or empty if a DPS is being used");
+                    }
+
+                    if (string.IsNullOrEmpty(this.DPSMasterSymmetricKey))
+                    {
+                        if (string.IsNullOrEmpty(this.DeviceIdentityCert) || !File.Exists(this.DeviceIdentityCert))
+                        {
+                            throw new ArgumentException("Device identity certificate path is invalid");
+                        }
+
+                        if (string.IsNullOrEmpty(this.DeviceIdentityPk) || !File.Exists(this.DeviceIdentityPk))
+                        {
+                            throw new ArgumentException("Device identity certificate path is invalid");
```

Comment:
Should this exception message refer to the "Device identity private key path" instead of "certificate path"?

Reply:
Yes it should :). Good catch.